helmfile/charts/networkpolicy-service/values.yaml

```diff
@@ -0,0 +1,36 @@
+harbor:
```

Comment:
Do we want to have a `additionalNetworkPolicies` so that we can add arbitrary NPs from config? Just in case we find some corner case and we can't wait until next version

Reply:
Yes i can add that. Do you want to be applied to all or do you want to have individual control?